encrypted-media-respec.html

```diff
@@ -1915,14 +1915,17 @@
 
         <dt>Promise&lt;void&gt; remove()</dt>
         <dd>
-          <p>Removes stored session data associated with this object.</p>
+          <p>
+            Removes all license(s) and key(s) associated with the session.
+            Other session data will be cleared as defined for each session type once a release message acknowledgment is processed by <a def-id="update"></a>.
```

Comment:
"For persistent sessions, ..." ?

If "persistent" is not sufficiently defined, we could link to https://w3c.github.io/encrypted-media/#is-persistent-session-type.